index.html

```diff
@@ -0,0 +1,6349 @@
+                    49,
+                    35,
+                    255
+                ],
+                [
+                    177,
+                    49,
+                    37,
+                    255
+                ],
+                [
+                    107,
+                    49,
+                    45,
+                    255
+                ],
+                [
+                    35,
+                    49,
+                    53,
+                    255
+                ],
+                [
+                    22,
+                    48,
+                    53,
+                    255
+                ],
+                [
+                    94,
+                    49,
+                    45,
+                    255
+                ],
+                [
+                    207,
+                    49,
+                    33,
+                    255
+                ],
+                [
+                    146,
+                    49,
+                    40,
+                    255
+                ],
+                [
+                    21,
+                    49,
+                    54,
+                    255
+                ],
+                [
+                    87,
+                    49,
+                    47,
+                    255
+                ],
+                [
+                    65,
+                    49,
+                    49,
+                    255
+                ],
+                [
+                    30,
+                    49,
+                    53,
+                    255
+                ],
+                [
+                    120,
+                    49,
+                    43,
+                    255
+                ],
+                [
+                    155,
+                    49,
+                    39,
+                    255
+                ],
+                [
+                    160,
+                    49,
+                    39,
+                    255
+                ],
+                [
+                    189,
+                    49,
+                    35,
+                    255
+                ],
+                [
+                    157,
+                    49,
+                    39,
+                    255
+                ],
+                [
+                    88,
+                    49,
+                    47,
+                    255
+                ],
+                [
+                    60,
+                    49,
+                    49,
+                    255
+                ],
+                [
+                    31,
+                    49,
+                    53,
+                    255
+                ],
+                [
+                    51,
+                    49,
+                    51,
+                    255
+                ],
+                [
+                    25,
+                    49,
+                    54,
+           
```

Comment:
global variables are evil
use [IIFE ](https://developer.mozilla.org/ru/docs/%D0%A1%D0%BB%D0%BE%D0%B2%D0%B0%D1%80%D1%8C/IIFE)to avoid it